common/src/test/java/com/revenuecat/purchases/common/testDurationExtensions.kt

```diff
@@ -0,0 +1,14 @@
+package com.revenuecat.purchases.common
+
+import com.revenuecat.purchases.utils.add
+import com.revenuecat.purchases.utils.subtract
+import java.util.Date
+import kotlin.time.Duration
+
+fun Duration.fromNow(): Date {
+    return Date().add(this)
+}
+
+fun Duration.ago(): Date {
+    return Date().subtract(this)
+}
```

Comment:
I thought about adding these extensions to production code, but since I wasn't planning to use them yet, I left them as test code for now. We can move them in the future if needed.